src/main/java/games/strategy/engine/framework/startup/launcher/ServerLauncher.java

```diff
@@ -131,15 +137,8 @@ protected void launchInNewThread(final Component parent) {
       }
       m_remoteMessenger.registerRemote(m_serverReady, ClientModel.CLIENT_READY_CHANNEL);
       m_gameData.doPreGameStartDataModifications(m_playerListing);
-      logger.fine("Starting server");
+      logger.info("Starting server");
       m_abortLaunch = testShouldWeAbort();
-      final byte[] gameDataAsBytes;
-      try {
-        gameDataAsBytes = gameDataToBytes(m_gameData);
```

Comment:
`gameDataToBytes()` is no longer referenced and can be removed.  (Note that the implementation of this method is what I was referring to above with respect to the new `GameData#toBytes()` method.)

Reply:
Thank you, that is a good cleanup - bcdd1af contains the update